vsts/pipelines/templates/_builderTemplate.yml

```diff
@@ -8,6 +8,13 @@ steps:
   displayName: 'Install Pack CLI'
   condition: true
 
+- task: Docker@1
+  displayName: Container registry logout
+  inputs:
+    command: logout
+    azureSubscriptionEndpoint: ${{ parameters.ascName }}
+    azureContainerRegistry: ${{ parameters.acrName }}
+
```

Comment:
this is not getting used but let's revert this?

Reply:
What code i removed in the pipeline.. now i added and commented.
But removing this piece of code.